Fix abac oplicy example file

CaoShuFeng opened the pull request:
"system:unauthenticated" is a group name rather than user name.
Fix it in the example file.

<!--  Thanks for sending a pull request!  Here are some tips for you:
1. If this is your first time, read our contributor guidelines https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md and developer guide https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md
2. If you want *faster* PR reviews, read how: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/faster_reviews.md
3. Follow the instructions for writing a release note: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes
-->

**Release note**:
<!--  Steps to write your release note:
1. Use the release-note-* labels to set the release note state (if you have access) 
2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`. 
-->
```NONE
```

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/42152)
<!-- Reviewable:end -->

CaoShuFeng committed:
Fix abac oplicy example file

"system:unauthenticated" is a group name rather than user name.
Fix it in the example file.

liggitt commented:
/lgtm

CaoShuFeng commented:
@k8s-bot non-cri e2e test this

liggitt commented:
@k8s-bot non-cri node e2e test this

CaoShuFeng commented:
@k8s-bot verify test this

liggitt commented:
@k8s-bot verify test this

CaoShuFeng commented:
@k8s-bot verify test this

liggitt commented:
@k8s-bot verify test this
